package.json

```diff
@@ -17,7 +17,7 @@
   ],
   "scripts": {
     "start": "./bin/server.js",
-    "test": "nodeunit"
+    "test": "jshint bin/ lib/ test/ *.js && nodeunit"
```

Comment:
Though I could add `"requireSpacesInsideParentheses": null` to the config to turn of that one rule and fix the rest.